ios/Classes/SwiftAdmobFlutterPlugin.swift

```diff
@@ -36,7 +36,12 @@ public class SwiftAdmobFlutterPlugin: NSObject, FlutterPlugin {
   public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
     switch call.method {
     case "initialize":
-        GADMobileAds.sharedInstance().start(completionHandler: nil)
+        // https://developers.google.com/admob/ios/test-ads#enable_test_devices
+        // **iOS simulators are automatically configured as test devices.**
+        // GADMobileAds.sharedInstance().requestConfiguration.testDeviceIdentifiers = ["You're physical iDevice Id"]
+        GADMobileAds.sharedInstance().start { (status: GADInitializationStatus) in
```

Comment:
`GADMobileAds.sharedInstance().requestConfiguration.testDeviceIdentifiers = [kGADSimulatorID as! String]` is not required. I've tested this out and the docs state __iOS simulators are automatically configured as test devices.__

Reply:
Note: If you do want to add a test device. You can uncomment line 41 and put in your device id. At least until this gets implemented on the flutter side.